test/integration/compile-api-blueprint-test.coffee

```diff
@@ -17,9 +18,10 @@ describe('compile() · API Blueprint', ->
     transactions = undefined
 
     beforeEach((done) ->
-      compileFixture(fixtures.missingTitleAnnotation.apiBlueprint, (args...) ->
-        [err, {warnings, transactions}] = args
-        done(err)
+      compileFixture(fixtures.missingTitleAnnotation.apiBlueprint, (err, results) ->
+        return done(err) if err
+        {warnings, transactions} = results
+        done()
```

Comment:
This change and all subsequent changes in this file was made because when there was an error and no results, constructs like

```coffee
[err, {warnings, transactions}] = args
```

were exploding.